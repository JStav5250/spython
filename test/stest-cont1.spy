for int i in range(20):
    continue

for int i in [1, 3, 5]:
    continue

for int j in [1, 3, 5]:
    if j == 4:
        continue


