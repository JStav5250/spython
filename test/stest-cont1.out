CodegenError: Continue has not been implemented for codegen!!
